<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dragModeButton">
        <property name="text">
         <string>Default (Ctrl + m)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addNodeButton">
        <property name="text">
         <string>Add Node (CTRL + n)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addEdgeButton">
        <property name="text">
         <string>Add Edge (Ctrl + e)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeButton">
        <property name="text">
         <string>Remove Item (Ctrl + r)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="currentText">
         <string>Select algorithm</string>
        </property>
        <item>
         <property name="text">
          <string>Select algorithm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DFS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BFS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Max Flow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dijkstra</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Kruskal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Euler cycle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hamiltonian cycle</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="hintLabel">
        <property name="styleSheet">
         <string notr="true">* {
	color: black;
	background-color: rgb(214, 255, 212);
	margin: 0;
	padding: 7px;
	font-size: 16px;
	border-style: solid;
	border-width: 1px;
	border-color: rgb(153, 255, 187)
}</string>
        </property>
        <property name="text">
         <string>Click on the workspace to add a vertex</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="stopAlgorithmButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="myres.qrc">
          <normaloff>:/images/images/stopBtn.png</normaloff>:/images/images/stopBtn.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="graphInfoLabel">
        <property name="text">
         <string>Graph info</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lastUpdatesLabel">
        <property name="text">
         <string>Added new edge</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="myres.qrc"/>
 </resources>
 <connections/>
</ui>
